website/server/controllers/api-v4/bug-report.js

```diff
@@ -0,0 +1,81 @@
+import nconf from 'nconf';
+import { authWithHeaders } from '../../middlewares/auth';
+import { convertVariableObjectToArray, sendTxn } from '../../libs/email';
+
+const api = {};
+
+/**
+ * @api {post} /api/v4/bug-report Report an issue
```

Comment:
todo test

Reply:
I tried an api-v4 integration but I can't get the `sendTxn` to "work" or I have no idea on how to check anything there (since the its not in the same process) 

 but I think this needs to be an unit test ... or atleast I'm trying it